src/components/pages/pages.pw.tsx

```diff
@@ -0,0 +1,194 @@
+          mount,
+          page,
+        }) => {
+          const props = { [propName]: propValue };
+
+          await mount(<PagesComponent {...props} />);
+
+          await dataComponentButtonByText(page, "Open Preview").click();
+
+          await expect(getDataElementByValue(page, "title")).toHaveText(
+            "My Second Page"
+          );
+        });
+      });
+    });
+  });
+
+  // TODO: Playwright is not returning the full class attribute, so this test is not working as expected.
+  (["slide", "fade"] as PagesProps["transition"][]).forEach((transition) => {
+    test.skip(`should render component with transition set to ${transition}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<PagesComponent transition={transition} />);
+
+      await dataComponentButtonByText(page, "Open Preview").click();
+      const pageComponent = getDataElementByValue(page, "visible-page");
+      await waitForAnimationEnd(pageComponent);
+
+      const classAttribute = pageComponent.getAttribute("class");
+      await expect(classAttribute).toContain(transition);
+    });
+  });
+
+  test("should render component and go next to Second page", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<PagesComponent />);
+
+    await dataComponentButtonByText(page, "Open Preview").click();
+    await dataComponentButtonByText(page, "Go to second page").click();
+
+    await expect(getDataElementByValue(page, "title").nth(0)).toHaveText(
+      "My Second Page"
+    );
+  });
+
+  test("should render component and go next to Third page", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<PagesComponent initialPageIndex={1} />);
+
+    await dataComponentButtonByText(page, "Open Preview").click();
+    await dataComponentButtonByText(page, "Go to third page").click();
+
+    await expect(getDataElementByValue(page, "title").nth(0)).toHaveText(
+      "My Third Page"
+    );
+  });
+
+  test("should render component and go back to Second page", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<PagesComponent initialPageIndex={2} />);
+
+    await dataComponentButtonByText(page, "Open Preview").click();
+    await backArrow(page).click();
+
+    await expect(getDataElementByValue(page, "title").nth(0)).toHaveText(
+      "My Second Page"
+    );
+  });
+
+  test("should render component and go back to First page", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<PagesComponent initialPageIndex={1} />);
+
+    await dataComponentButtonByText(page, "Open Preview").click();
+    await backArrow(page).click();
+
+    await expect(getDataElementByValue(page, "title").nth(0)).toHaveText(
+      "My First Page"
+    );
+  });
+
+  [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS].forEach(
+    ([testTitle]) => {
+      test(`Component should render as expected with title prop set to ${testTitle}`, async ({
```

Comment:
```suggestion
      test(`Page Component should render as expected with title prop set to ${testTitle}`, async ({
```

**suggestion** (non-blocking): while I don't think this needs to be in its own `describe` block as it was in the Cypress tests, I think it's maybe worth highlighting that this test is specifically about the Page (sub)component